src/applications/vaos/utils/selectors.js

```diff
@@ -201,12 +209,9 @@ export function getDateTimeSelect(state, pageKey) {
   const eligibilityStatus = getEligibilityStatus(state);
   const systemId = getSiteIdForChosenFacility(state);
 
-  const availableDates = availableSlots?.reduce((acc, s) => {
-    if (!acc.includes(s.date)) {
-      acc.push(s.date);
-    }
-    return acc;
-  }, []);
+  const availableDates = Array.from(
+    new Set(availableSlots?.map(slot => slot.start.split('T')[0])),
```

Comment:
I think this is faster, and hopefully clearer. Though we might not have enough slots for the performance to matter.